qa/integration/specs/cli/prepare_offline_pack_spec.rb

```diff
@@ -3,15 +3,8 @@
 require_relative "../../framework/settings"
 require_relative "../../services/logstash_service"
 require_relative "../../framework/helpers"
-require "logstash/devutils/rspec/spec_helper"
 
-
-# These are segmented into a separate tag that MUST be run separately from any docker tests
-# The reason they break the Docker API and that in turn even breaks tests not using Docker 
-# is that the Docker API has a global singleton Docker container set up as soon as it's 
-# required that acts in the background and will err out if the internet is down
-# See https://github.com/elastic/logstash/issues/7160#issue-229902725
-describe "CLI > logstash-plugin prepare-offline-pack", :offline => true do
+describe "CLI > logstash-plugin prepare-offline-pack" do
```

Comment:
They were only separated because they broke the Docker dependencies, which no longer exist (as the delete comment suggests).

Reply:
I don't think that's correct.

I'm the one who separated the tests IIRC, and the issue, if I recall, was that the special C wrapper used to go 'offline' would break the non-offline tests.

Reply:
spoke on slack, and believe we came to a consensus that this change is OK since it only caused issues with the Docker API which we don't use anymore.  @andrewvc - can you confirm ?